docs/user/index.rst

```diff
@@ -2,9 +2,14 @@
 User documentation
```

Comment:
We should not name the same thing with different names.
`User documentation` must be replaced by `Information` or `User Information` (once again, `User Help` is my preference)